General: Widescreen fixes

DarthMDev opened the pull request:
This pr attempts to fix some widescreen issues

Also I am aware of the astrond commit I will remove this commit when my pr is ready

I needed it to run the game on macos m1

Should fix #59

DarthMDev committed:
Astrond: Add universal2 astrond

This also is updated to have builtin mongo support

DarthMDev committed:
PetObserve: Fix IntEnum

Update from pythonutil enum

DarthMDev committed:
Merge pull request #3 from Toonfisch/bugfix

DarthMDev committed:
Merge remote-tracking branch 'upstream/develop' into develop

DarthMDev committed:
Merge branch 'open-toontown:develop' into widescreen-fixes

DarthMDev committed:
General: Widescreen fixes

chat, cannon, racing, boarding groups, quests,  catalog,  and elevator widescreen fixes
Some widescreen fixes to battle , battles in building, battles in levels , the catalog screen, and the chat
Also add some resolutions to settings

Friends: Fix widescreen issues on the friend section

PAT and MAT,
minigames, gag shop after trolley,
shticker book,
friends list , laff meter
avatar details panel
and teleport panel widescreen fixes

General: More widescreen fixes

Some fixes for OTPTimer, and the furniture gui
And another battle thing

Catalog: Fix catalog notification dialog for widescreen

Some more widescreen fixes

Kart: Fix crash when buying kart

ToontownBattleGlobals: Sorta fix FaceOffFov

MichaelGDev48 commented:
todo for me to fix :
<img width="1276" alt="image" src="https://user-images.githubusercontent.com/10155689/213604307-4c308bf6-6ef1-492b-81a3-a999f2d94353.png">

MichaelGDev48 commented:
another issue
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/10155689/213604672-a8a7891a-6cff-4f77-9c55-110b91600db5.png">

MichaelGDev48 commented:
Toon escape too
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/10155689/213604886-589dbf37-a9f6-47c7-a497-477fc3e93bd9.png">

MichaelGDev48 commented:
Maze game: Can see the gray  at the beginning

DarthMDev commented:
Anyone is welcome to help test the last issues seems to be the minigames

MichaelGDev48 commented:
Ngl I never experienced this before I didn't have these issues when I first implemented this in anesidora so this is very strange 🤔

DarthMDev committed:
Minigames: Fix issues with the gray showing

CogThief: Fix gameboard being too small for widescreen

DistributedVineGame: Fix widescreen for background

Fix ice game widescreen issue

Minigames: Fix widescreen issues

This is a better fix for cogthief and jungle vines

So the player doesn't see the void

TwoD: Fix widescreen issue with the 2d minigame

MazeGame:  mazegame fix for the gray

DarthMDev commented:
Fixed the minigames issues with the gray showing

DarthMDev commented:
Ok I think I resolved everything widescreen wise 
(I reverted  the astrond commit although the commit msg itself remains) 
@LittleToonCat mind taking a look?

hunterboppen commented:
darth still waiting for the commit to be pulled

DarthMDev committed:
Merge branch 'develop' into widescreen-fixes

freshollie commented:
Any reason for this not to be merged?

DarthMDev committed:
Merge branch 'open-toontown:develop' into widescreen-fixes
